contracts/truefi2/TrueRateAdjuster.sol

```diff
@@ -222,16 +222,16 @@ contract TrueRateAdjuster is ITrueRateAdjuster, UpgradeableClaimable {
         }
         uint256 coefficient = uint256(creditScoreRateConfig.coefficient);
         uint256 power = uint256(creditScoreRateConfig.power);
-        return min(coefficient.mul(MAX_CREDIT_SCORE**power).div(score**power).sub(coefficient), MAX_RATE_CAP);
+        return min(coefficient.mul(uint256(MAX_CREDIT_SCORE)**power).div(uint256(score)**power).sub(coefficient), MAX_RATE_CAP);
```

Comment:
I'm guessing this fixes the build warning? Thanks! Ideally we'd split this to a separate PR but I'll let it slide here.